submissions/JuliaVorotchenko/Popup-task/index.html

```diff
@@ -0,0 +1,143 @@
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">YouTube</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Messenger</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Photos</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Calendar</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Translate</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Weather</span>
+                                    </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Notes</span>
+                                     </a>
+                            </li>
+                            <li class="app-list-item">
+                                    <a href="#" class="app-link">
+                                        <span class="app-icon"></span>
+                                        <span class="app-link-name">Account</span>
+                                    </a>
+                            </li>
+                        </ul>
+                        <input class="popup-input" type="checkbox" id="show-hidden-apps" name="more_apps" value="">
```

Comment:
```.popup-input:checked``` could be used for:
1. Open and close section More
2. Hide and show the more-button

I mean: do not use ```.popup-input:focus ~ .hidden-apps```